### PR TITLE
DevKit updates for master branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

**Comment by OskarStark** [comment]
Travis failed, I restarted the build

**Comment by OskarStark** [comment]
Still failing, can you check @greg0ire ?

**Comment by greg0ire** [comment]
Same problem as in my other PR I guess :/ will check later.

File: .travis.yml [hunk 1]
```diff
@@ -40,9 +37,13 @@ matrix:
...
     - php: '7.0'
       env: TARGET=lint
-    - php: '5.3'
+    - php: '5.6'
       env: COMPOSER_FLAGS="--prefer-lowest"
     - php: '5.6'
+      env: SYMFONY=2.3.*
+    - php: '5.6'
+      env: SYMFONY=2.7.*
+    - php: '5.6'
```
Comment (greg0ire) [inline .travis.yml]: @OskarStark wait why are we adding that back?

**Comment by greg0ire** [comment]
Should have been done in https://github.com/sonata-project/dev-kit/pull/154

**Pull request closed without merging**